app/src/main/java/org/fossasia/openevent/data/Version.java

```diff
@@ -0,0 +1,22 @@
+package org.fossasia.openevent.data;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Created by MananWason on 05-06-2015.
+ */
+public class Version {
+    @SerializedName("event_id")
+    int eventId;
+    @SerializedName("event_ver")
+    int eventVer;
+    int id;
```

Comment:
@creativepsyco The version id will be the primary check. If version id is changed then there is same change in the data.